### DISCUSSION TITLE
Why is `Link.to` typed as string?

### DISCUSSION THREAD

**Discussion by 1EDExg0ffyXfTEqdIUAYNZGnCeajIxMWd2vaQeP** [zoontek/chicane#32] [Q&A] [answered]
It's very easy to accidentally misuse when coming from react-router and typing `<Link to={`contacts/${contact.id}`}>` without thinking.

**Comment by zoontek** [comment, marked as answer]
For external links and because it can be quite hard to type Link props on user side (if you want to implement your own Link component using `useLinkProps`

Just use `<Link to={Router.YourRouteName()}>`
